Extensions/AdMob/JsExtension.js

```diff
@@ -93,6 +93,41 @@ module.exports = {
       .setIncludeFile('Extensions/AdMob/admobtools.js')
       .setFunctionName('gdjs.adMob.setTestMode');
 
+    extension
+      .addAction(
+        'PreventAdmobAutoInitialization',
+        _('Prevent Admob auto initialization'),
+        _(
+          'Prevent Admob from initializing automatically. You will need to call InitAdmob manually.\n' +
+            'This is useful if you want to control when the consent dialog will be shown (for example, after the user has accepted your game terms).'
+        ),
+        _('Prevent Admob auto initialization'),
+        '',
+        'JsPlatform/Extensions/admobicon24.png',
+        'JsPlatform/Extensions/admobicon16.png'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/AdMob/admobtools.js')
+      .setFunctionName('gdjs.adMob.preventAdmobAutoInitialization');
+
+    extension
+      .addAction(
+        'InitializeAdmob',
+        _('Initialize Admob'),
```

Comment:
Everywhere for user facing texts: AdMob

(keep Admob for code, because it's camelCase and we consider "admob" as a single word)